config/claude/pushover.sh

```
@@ -0,0 +1,104 @@
+            send_notification "🔐 ${TOOL} permission required" 1
+            ;;
+        *)
+            send_notification "ℹ️ ${MESSAGE}" 0
+            ;;
+    esac
+    exit 0
+fi
+
+# Handle Stop hook (session completion)
+if echo "$input" | jq -e '.session_id' >/dev/null 2>&1; then
+    SESSION_ID=$(echo "$input" | jq -r '.session_id[0:8]')
```

Shellcheck (0.11.0)

[warning] SESSION_ID appears unused. Verify use (or export if used externally).

(SC2034)